src/status_im/ui2/screens/chat/actions.cljs

```diff
@@ -52,66 +58,199 @@
          (if muted?
            #(unmute-chat-action chat-id)
            #(mute-chat-action chat-id))
-         false))
+         false
+         nil
+         true))
 
 (defn mark-as-read-entry [chat-id]
-  (entry :main-icons2/check
+  (entry :main-icons2/correct
          (i18n/label :mark-as-read)
          #(mark-all-read-action chat-id)
+         false
+         nil
          false))
 
 (defn clear-history-entry [chat-id]
   (entry :main-icons2/delete
          (i18n/label :clear-history)
          #(clear-history-action chat-id)
-         true))
+         true
+         nil
+         false))
 
 (defn delete-chat-entry [chat-id]
   (entry :main-icons2/delete
          (i18n/label :delete-chat)
          #(delete-chat-action chat-id)
-         true))
+         true
+         nil
+         false))
+
+(defn leave-group-entry [chat-id]
+  (entry :main-icons2/log-out
+         (i18n/label :leave-group)
+         #(leave-group-action chat-id)
+         true
+         nil
+         false))
 
 (defn view-profile-entry [chat-id]
   (entry :main-icons2/friend
          (i18n/label :view-profile)
          #(show-profile-action chat-id)
+         false
+         nil
          false))
 
 (defn edit-nickname-entry [chat-id]
   (entry :main-icons2/edit
          (i18n/label :edit-nickname)
          #(edit-nickname-action chat-id)
+         false
+         nil
+         false))
+
+(defn notifications-entry []
+  (entry :main-icons2/notifications
+         (i18n/label :notifications)
+         #(js/alert "TODO: to be implemented, requires design input")
+         false
+         "All messages" ; placeholder
```

Comment:
should "placeholder" be a TODO comment? 

or should "All messages" be an i18n label?